### DISCUSSION TITLE
How to use hexahedral elements and keep the shape of grains unchanged？

### DISCUSSION THREAD

**Discussion by smallrain1996** [neperfepx/neper#210] [Q&A] [answered]
In the 3D model of the -M module, how to use irregular hexahedral elements to ensure the grain shape? 
In other words, I hope to use C3D8 units (for ABAUQS), but I also hope that the grain boundaries are still flat and not jagged.
Thank you very much.

**Comment by rquey** [comment, marked as answer]
See #98.
